acceptance/framework/consul/cli_cluster.go

```diff
@@ -0,0 +1,246 @@
+package consul
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	terratestk8s "github.com/gruntwork-io/terratest/modules/k8s"
+	terratestLogger "github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/config"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/environment"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/helpers"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/k8s"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/logger"
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// CLICluster
+type CLICluster struct {
+	ctx                environment.TestContext
+	namespace          string
+	helmOptions        *helm.Options
+	values             map[string]string
+	releaseName        string
+	kubernetesClient   kubernetes.Interface
+	kubeConfig         string
+	kubeContext        string
+	noCleanupOnFailure bool
+	debugDirectory     string
+	logger             terratestLogger.TestLogger
+}
+
+func NewCLICluster(
```

Comment:
NewCLICluster is pretty similar to NewHelmCluster, with just a few more values being added to the struct that gets created.